### PR TITLE
Flight sim support & bug fixes

### PR DESCRIPTION
**Pull request by SilentSys**
Added support for the full flight sim tests and fixed some bugs that I found along the way,

### CODE REVIEW

**Comment by SilentSys** [comment]
[Altitude eqn is for Pa but we're using mbar](https://trello.com/c/rEpY7xHJ/326-altitude-eqn-is-for-pa-but-were-using-mbar)

**Comment by SilentSys** [comment]
[Instability in single sensor SIM integration tests](https://trello.com/c/soj1F9FX/334-instability-in-single-sensor-sim-integration-tests)

**Comment by SilentSys** [comment]
[State machine goes into LANDED immediately after main is deployed](https://trello.com/c/BpWjPCAg/327-state-machine-goes-into-landed-immediately-after-main-is-deployed)

File: avionics/common/include/radio.h [hunk 1]
```diff
@@ -69,6 +69,8 @@ class Radio {
     constexpr static Hal::ms WATCHDOG_SEND_INTERVAL{3000};
     constexpr static fwd_cmd_t STOP_PARSE_FLAG = 1 << 0;
     constexpr static fwd_cmd_t CAN_SEND_FLAG = 1 << 1;
+    
+    static bool m_can_send; // Is XBee available to send
```
Comment (akoen) [inline avionics/common/include/radio.h]: On FLARE we seem to have standardized around using a trailing _ to indicate member vars e.g. can_send_

**Review by akoen — DISMISSED** [review]

File: avionics/common/include/radio.h (hunk 1 above)
Comment (SilentSys) [reply avionics/common/include/radio.h]: Good catch!

File: avionics/common/lib/CalcHelpers/barometric.cpp [hunk 2]
```diff
@@ -6,13 +6,15 @@
  * @param pascals Pressure, in [Pa]
  * @return Altitude above sea level [m]
  */
-float pressureToAltitude(float const pres) {
+float /* m */ pressureToAltitude(float pres /* mbar */) {
```
Comment (jesseli2002) [inline avionics/common/lib/CalcHelpers/barometric.cpp]: there's a documentation section right above the function that lists units, and also commit 8da79407 updates the barometer sensor object to return Pa instead (my bad, I should've updated you beforehand)

note this also means below changes to tests should be reverted

**Review by jesseli2002 — CHANGES_REQUESTED** [review]

File: avionics/common/lib/CalcHelpers/barometric.cpp (hunk 2 above)
Comment (SilentSys) [reply avionics/common/lib/CalcHelpers/barometric.cpp]: So with those changes we use Pa everywhere? So should be changed in the GS/SIM as well?

Comment (jesseli2002) [reply avionics/common/lib/CalcHelpers/barometric.cpp]: yeah, probably - it's probably best to use SI base units wherever possible

**Review by jesseli2002 — APPROVED** [review]

**Pull request merged**